src/server/cluster/cluster_config.h

```diff
@@ -19,6 +19,7 @@ enum class MigrationState : uint8_t {
   C_CONNECTING,
   C_SYNC,
   C_FINISHED,
+  C_CANCELLED,
```

Comment:
why do you need it? If we cancel migration we should remove it at all

Reply:
See the usage near calling `FinishMigration()`, although I'm open to suggestions here :)